Port to Qt5

dschmidt opened the pull request:
Hi,

this allows building QtSparkle with Qt4 and Qt5.
I bumped the CMake version requirement to keep the changes minimal (Qt5 target support, Automoc, Autouic).


Regards,
Domme

davidsansome commented on CMakeLists.txt:
```diff
@@ -1,10 +1,31 @@
-include(${QT_USE_FILE})
+option(BUILD_WITH_QT4 "Build qtsparkle with Qt4" ON)
+if(BUILD_WITH_QT4)
+    find_package(Qt4 REQUIRED QtCore QtGui QtNetwork QtXml)
```
Can you use 2-space indentation here to match the convention elsewhere?

dschmidt replied:
Sure, sorry, was already late :)

davidsansome commented on src/CMakeLists.txt:
```diff
@@ -86,11 +69,16 @@ compile_translations(SOURCES
 )
 
+set_target_properties(qtsparkle
+    PROPERTIES
```
Here too

davidsansome commented:
Thanks so much for doing this!

dschmidt commented:
Fixed intendation, made Qt4 and Qt5 version completely coinstallable by also suffixing the include dir and removed some left over references to ${MOC_SOURCES} ${UI_SOURCES}.
